### PR TITLE
include short alarm name in slack notification text

### PR DESCRIPTION
**Pull request by doodlemoonch**
Provide a bit of context to notifications by including the shortname in the notification text

### CODE REVIEW

File: functions/notify_slack.py [hunk 1]
```diff
@@ -60,8 +60,9 @@ def notify_slack(message, region):
         "attachments": []
     }
     if "AlarmName" in message:
-        payload['text'] = "AWS CloudWatch notification"
-        payload['attachments'].append(cloudwatch_notification(message, region))
+        notification = cloudwatch_notification(message, region)
+        payload['text'] = "AWS CloudWatch notification - " + notification['fields'][0]['value']
```
Comment (antonbabenko) [inline functions/notify_slack.py]: Maybe replace `notification['fields'][0]['value']` with something what comes from SNS message already - `message["AlarmName"]` ?

**Comment by doodlemoonch** [comment]
@antonbabenko done

**Comment by antonbabenko** [comment]
Thanks, Matt!

[v1.8.0](https://github.com/terraform-aws-modules/terraform-aws-notify-slack/releases/v1.8.0) has been released

**Comment by doodlemoonch** [comment]
@antonbabenko thanks!

**Pull request merged**